docs/maintain/iota-core/2.0/docs/how_tos/validator_setup.md

```diff
@@ -0,0 +1,345 @@
+command `echo {Public Key}` means replacing the variable `{Public Key}` with its value, i.e., running the
+command `echo 0xff`.
+
+:::
+
+## Set Up the CLI Wallet
+
+You can download the latest release of the CLI Wallet from the following link:
+https://github.com/iotaledger/iota-sdk/releases/tag/cli-wallet-v2.0.0-alpha.1
+
+After downloading, copy the binary to your current directory.
+
+### Build From Source
+
+Please follow the instructions to build the CLI Wallet from source:
+
+#### 1. Clone the Repository
+
+You can clone the repository by running the following command:
+
+```bash
+git clone -b 2.0 https://github.com/iotaledger/iota-sdk
+```
+
+#### 2. Build
+
+After you have downloaded the source code, you can build it by changing the current directory to `iota-sdk/cli` and
+running the following command:
+
+```bash 
+cargo build --release
+```
+
+#### 3. (optional) Copy the Wallet
+
+For ease of access, you can copy the Wallet that you built into your current directory or even add it to your $PATH:
+
+```bash
+cp ../target/release/wallet wallet
+```
+
+#### 4. Connect to a Node
+
+You can use the following command to connect your wallet to a Testnet node — alternatively, your node’s URL:
+
+```bash
+./wallet init --node-url "https://api.nova-testnet.iotaledger.net"
+```
+
+#### 5. Configure Your Wallet
+
+Once you’ve connected your wallet to a node, you must answer the following setup questions:
+
+##### Select secret manager.
+
+Unless you have a ledger, select Stronghold.
+
+##### Do you want to set the address of the new wallet?
+
+No.
+
+##### Select bip path
+
+Select any bip path of your choice (e.g., IOTA is fine)
+
+##### Do you want to set an alias for the new wallet?
+
+Yes.
+
+Then, enter a wallet alias of your choice.
+
+## Create an Account
+
+:::tip Connect to a Node
+
+The following sections assume you have already set up your [CLI Wallet](#set-up-the-cli-wallet), 
+and [connected it to a node](#4-connect-to-a-node).
+
+:::
+
+### 1. Create an Implicit Account
+
+You can create an implicit account by running the following command:
+
+```
+implicit-account-creation-address
+``` 
+
+Please copy the implicit address returned by the command.
+
+### 2. Fund Your Account
+
+You can enter the address at the [Testnet Faucet](https://faucet.nova-testnet.iotaledger.net/) to fund your account or run the
+following command
+
+```bash
+faucet {implicit-address} --url https://faucet.nova-testnet.iotaledger.net/api/enqueue
+```
+
+### 3. Sync With the Node
+
+After creating and funding your implicit account, you should use the `sync` and `implicit-accounts` functionalities to
+retrieve the implicit account creation output.
+
+```bash 
+sync
+implicit-accounts
+```
+
+You should run this until the BIC changes from `None` to `Some(0)`, then copy the `Output Id`.
+
+### 4. Transition the Account
+
+You can transition your account by running the following command:
+
+```bash
+./wallt implicit-account-transition {Output Id}
```

Comment:
remove `./wallt/` since this is an internal wallet command

Reply:
```suggestion
implicit-account-transition {Output Id}
```